index.js

```diff
@@ -15,11 +15,13 @@
 const { AssetCompute } = require("./lib/assetcompute");
 const { AssetComputeEventEmitter } = require("./lib/eventemitter");
 const { AssetComputeClient } = require("./lib/client");
+const { AssetComputeClientRetry } = require("./lib/client-retry");
```

Comment:
let me know if you prefer a different name, something that clearly lets the user know this is related to retries on 429s: `AssetComputeClient429Retry`

Reply:
No, in case we ever need to extend. This name is fine!